Adding step to remind users to fetch all lfs objects;

daerogami opened the pull request:
Addresses issue #3

daerogami committed:
Adding step to remind users to fetch all lfs objects;

dkonar commented on CONTRIBUTING.md:
````diff
@@ -18,6 +18,12 @@ Thank you for considering contributing to our project! Here are a few guidelines
+    ```sh
+    git lfs fetch --all
+    ```
+
````
Right now `git lfs fetch --all` is ok because we don't have maybe branches but [we could run in to issues later](https://stackoverflow.com/questions/72610494/what-is-the-difference-between-git-lfs-fetch-git-lfs-fetch-all-and-git/72610495#72610495) if we have a lot of asset revisions on different branches. 

Can we also add a link to https://git-lfs.com/ in the prerequisites section of the README, since `git-lfs` needs to be installed?

daerogami committed:
Updated example user's link to use same repo name;
Updated lfs example to use pull without --all;
Added reference to development reference files;
